index.html

```diff
@@ -30,20 +30,20 @@ <h1>Privacy Community Group</h1>
 title="Community Group">CG</abbr></a>
 is <a href=charter.html>chartered</a> <q cite=charter.html#mission>to
 incubate privacy-focused web features and APIs to improve user privacy
-on the web through enhanced browser behavior.</q> If that sounds
-interesting to you, please
-<a href=https://www.w3.org/community/privacycg/join>join us</a>!
-(In order to join, you'll need to
-<a href=https://www.w3.org/accounts/request>request a W3C account</a>
-if you don't already have one.)
+on the web through enhanced browser behavior.</q>
 
-<p>If you have any questions or concerns, the
-<a href=charter.html#chairs>chairs</a> are more than happy to help. The
-easiest way to get in touch with them is to
-<a href=https://github.com/privacycg/admin/issues/new>file an issue</a>
-(preferred) or to
-simply <a href=mailto:erik.anderson@microsoft.com;hober@apple.com;tanvi@mozilla.com>email
-them</a> (if your concern can’t be raised publicly).
+<h2>Participate</h2>
+
+<p>Our <a href=https://www.w3.org/community/privacycg/>W3C page</a>
+contains a list of
+<a href=https://www.w3.org/community/privacycg/participants>current
+participants</a>.
+
+<p>We hope you'll
```

Comment:
Switch this with the previous paragraph; first asking folks to participate, then showing the current participants.

Reply:
May require a change to the beginning of this sentence as well.

Reply:
Done.